charts/pomerium/templates/authorize-deployment.yaml

```diff
@@ -54,10 +56,9 @@ spec:
         image: {{ .Values.image.repository }}:{{ .Values.image.tag }}
         imagePullPolicy: {{ .Values.image.pullPolicy }}
         args:
-{{- if or .Values.config.existingConfig .Values.config.policy }}
+        {{- if or (or .Values.config.existingConfig .Values.config.policy) .Values.operator.enabled }}	
           - --config=/etc/pomerium/config.yaml
-{{- end }}
-{{- range $key, $value := .Values.extraArgs }}
+        {{- end }}{{- range $key, $value := .Values.extraArgs }}
```

Comment:
nit:  I think the for loop pops out a bit more on a seperate line

Reply:
Good catch.